config/locales/en.yml

```diff
@@ -237,13 +245,10 @@ en:
           link_text:
             sign_in: Sign in
             sign_up: sign up
+        title: Advertise a school job
       title: Find a job in teaching
-    signed_in_jobseeker:
-      description: Apply for jobs you are interested in, and check the status of your applications.
-      title: Apply for jobs
-    signed_out_jobseeker:
-      description_html: "%{link} or create an account to apply for jobs that use the Teaching Vacancies application form."
-      title: Apply for jobs
+      title_h1: Find teacher and school jobs near you
```

Comment:
Could `title` and `subtitle` be better?

Reply:
Maybe. I like that these title are grouped together and are easy to find starting all with title, and the HTML tag they are pertinent to